chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -198,6 +212,92 @@ impl pallet_ethereum_token_transfers::Config for Runtime {
     type WeightInfo = crate::weights::pallet_ethereum_token_transfers::SubstrateWeight<Runtime>;
 }
 
+/// `TokenTransferMessageProcessor` is responsible for receiving and processing native tokens
+/// sent from Ethereum. If the message is valid, it performs the token transfer
+/// from the Ethereum sovereign account to the specified destination account.
+pub struct TokenTransferMessageProcessor<T>(sp_std::marker::PhantomData<T>);
+impl<T> MessageProcessor for TokenTransferMessageProcessor<T>
+where
+    T: snowbridge_pallet_inbound_queue::Config + pallet_ethereum_token_transfers::Config,
+    T::AccountId: From<[u8; 32]>,
+{
+    fn can_process_message(channel: &Channel, envelope: &Envelope) -> bool {
```

Comment:
Where is can_process_message called? This is important as I dont think its being called in the pllaeT:
https://github.com/moondance-labs/polkadot-sdk/blob/tanssi-polkadot-stable2409/bridges/snowbridge/pallets/inbound-queue/src/lib.rs

Reply:
I am saying thgis because there are checks you do here that you dont do in process_message. And we need to make sure we check those

Reply:
I see it's in the tuple macro: https://github.com/moondance-labs/polkadot-sdk/blob/0e6fe9876480486026d065cb257321bd629db95c/bridges/snowbridge/primitives/router/src/inbound/mod.rs#L495

But as I said let's test this stuff